### ISSUE TITLE
APs is relatively weaker than APm and APl , I have some thoughts to improve this.

### ISSUE THREAD

**Issue by kartik4949** [google/automl#819]
so we can improve the APs by Overlapping Tiling , i.e  making a single image into multi smaller grids e.g 3x2 and treating indiviual tile/grid as single ground truth image , this eventually can increase object to image ratio, hence smaller object wont be smaller no more , 
but we have  to carefully pick the grid, so that large object are not very large.

@LucasSloan @mingxingtan @fsx950223 @NikZak pls give your thoughts.
Thanks

**Comment by mingxingtan** [comment]
Hi @kartik4949, interesting idea. I think it would help the training, but how to do the inference after the training?
If we still make each image to multiple tiles, the inference might be slow?

**Comment by kartik4949** [comment]
@mingxingtan  Yes inference will be slow but i can make separate process for each tile and will invoke them with async , so overall they only take time of one tile!

**Comment by kartik4949** [comment]
@mingxingtan and for inference we need to infere on each tile i.e we can have infere in batch of 4 , if we divide our image into 4 grid.
Next we can merge bbox of overlapping area with NMS

**Comment by JK-Pair** [comment]
Are there researches about something like this? @kartik4949  I'm interested in overlapping tiling, It will help in my project.

**Comment by kartik4949** [comment]
@JK-Pair yes search for power of tilling paper
